src/Dict.h

```diff
@@ -1648,6 +1672,9 @@ template <typename T> class Dictionary
 	uint32_t num_entries = 0;
 	uint32_t max_entries = 0;
 	uint64_t cum_entries = 0;
+	int64_t space_distance_samples = 0;
```

Comment:
They can probably be unsigned. I'm tempted to say that 32bit is probably enough and overflows are not a correctness issue, too. But maybe the 8bytes aren't worth the discussion ;-)